### ISSUE TITLE
[FEAT] Integrate with Google Health Connect

### ISSUE THREAD

**Issue by edersong** [joaovitoriasilva/endurain#35]
Integrate with Google Health Connect to get health data and to store it locally.

**Comment by joaovitoriasilva** [comment]
Agree, however Google Health has the same issue for me of Apple Health. To use the SDK a native app needs to exist and then it can export to Endurain. And currently I don’t have the skills to do it. If someone would like to contribute on this it would be great. 
For Apple Health I was thinking in the future have the ability for a user to upload the export files generated by Apple Health and bulk import that. Not perfect because it is manual, but it is something. I believe Google should have a similar export function.

**Comment by ThreeCO** [comment]
Some time ago I've been building two Shortcuts on my iPhone in order to ex-/import some of my health-data

1) Gather Health-Data (weight, HR-Variability etc.) per Day and send it to an API (base64encoded passwort,...)
2) Create a "Share-Sheet" in order to send FIT/GPX-Files from the App "HealthFit" to the same API.
Since i could not export FIT-Files from AppleHealth/AppleTraining I used this workaround. 

These Shortcuts are easily built - but right now I don't understand how to use the API or what features are supported ;-)

**Comment by joaovitoriasilva** [comment]
[!IMPORTANT]
This repository is being archived and the project has moved to Codeberg.
👉 New home: https://codeberg.org/endurain-project
Please open any new issues, pull requests, or discussions there. This GitHub repository will become read-only shortly.
Thank you for your contributions! 🙏
